### PR TITLE
I'm probably crazy but I just want it to work :(

### PR DESCRIPTION
**Pull request by bobby29831**
pls work idk ugh maybe this fixes who knows I definitely don't

im in a fit of delusion like I always am when I commit to this

merge at your own risk I don't claim responsibility

awoogah

wow wow wee wow

### CODE REVIEW

File: src/listeners/bumpNotifications.ts [hunk 1]
```diff
@@ -24,10 +22,10 @@ export const bumpNotificationListener: Listener = (client) => {
 	const init = async () => {
 		const botChannel = await client.channels.fetch(config.channels.botCommands) as TextChannel
 
-		const lastMessage = botChannel.messages.cache.find(successPredicate)
-		const delay = 7200 * 1000 + (lastMessage?.createdTimestamp || Date.now()) - Date.now()
+		const lastMessage = botChannel.messages.cache.find(bumpPredicate)
+		const delay = twoHours + (lastMessage?.createdTimestamp || Date.now()) - Date.now()
 		logger.info(`Next bump due in ${delay / 60000} minutes`)
-		setTimeout(sendBumpMessage, delay)
+		setTimeout(sendBumpMessage, (delay < 0 ? 0 : delay))
```
Comment (lucypoulton) [inline src/listeners/bumpNotifications.ts:28]:
```suggestion
		setTimeout(sendBumpMessage, Math.abs(delay))
```

**Review by lucypoulton — COMMENTED** [review]
to be honest I don't really see how this would fix anything

**Comment by bobby29831** [comment]
I don't either, but I don't get what's wrong with it in the first place... my thought process was that somehow the predicate was messing it up, but honestly it makes no sense why it isn't working I'm just sad :)

**Comment by bobby29831** [comment]
wait nvm was just thinking about it and changing the predicate to remove the timing part wouldn't help since it'd mess up checking if the bump was actually successful oopsies at least I caught my issues this time lol

**Pull request closed without merging**